example/cmd/device-simple/Dockerfile

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (c) 2018, 2019 Intel
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+FROM golang:1.11-alpine AS builder
+
+ENV GO111MODULE=on
+WORKDIR /go/src/github.com/edgexfoundry/device-sdk-go
+
+LABEL license='SPDX-License-Identifier: Apache-2.0' \
+  copyright='Copyright (c) 2018, 2019: Intel'
+
+RUN sed -e 's/dl-cdn[.]alpinelinux.org/nl.alpinelinux.org/g' -i~ /etc/apk/repositories
+
+# add git for go modules
+RUN apk update && apk add make git
+
+COPY go.mod .
+
+RUN go mod download
+
+COPY . .
+
+RUN make build
+
+# Next image - Copy built Go binary into new workspace
+FROM scratch
+
+LABEL license='SPDX-License-Identifier: Apache-2.0' \
+  copyright='Copyright (c) 2018, 2019: Intel'
+
+ENV APP_PORT=49990
+#expose command data port
+EXPOSE $APP_PORT
+
+WORKDIR /
+COPY --from=builder /go/src/github.com/edgexfoundry/device-sdk-go/example/cmd/device-simple/device-simple /usr/local/bin/device-simple
+COPY --from=builder /go/src/github.com/edgexfoundry/device-sdk-go/example/cmd/device-simple/res/docker/configuration.toml /res/docker/configuration.toml
```

Comment:
@lenny-intel  I notice we are using a separate copy of configuration.toml when building docker image.  Does it make sense to parameterize docker-specific values (to reduce maintenance)?

Also just be aware the container will run an additional two schedules compared to local build of device-simple. readTemperature and readHumidity events are currently instrumented only when running device-simple from container.

Reply:
@tobiasmo1 , this is the standard pattern used by all the services. Noted on the schedules.

Reply:
Gotcha. I noticed the duplication but It didn't look very straightforward to remedy.